src/commands.js

```diff
@@ -409,3 +409,26 @@ export function deleteTable(state, dispatch) {
   }
   return false
 }
+
+// :: (number, number, ?Object) → (EditorState, dispatch: ?(tr: Transaction)) → bool
+// Creates a table with specified size and attrs, then focus to first cell.
+export function createTable(row, col, cellAttrs = {}) {
+  return function(state, dispatch) {
+    const { tr, schema } = state
+    const tableType = schema.nodes.table
+    const rowType = schema.nodes.table_row
+    const cellType = schema.nodes.table_cell
+    const cellNode = cellType.createAndFill(cellAttrs)
+    const cells = []
+    for (let i = 0; i < col; i++) cells.push(cellNode)
+    const rowNode = rowType.create(null, Fragment.from(cells))
+    const rows = []
+    for (let i = 0; i < row; i++) rows.push(rowNode)
+    const tableNode = tableType.create(null, Fragment.from(rows))
+    const newSelection = TextSelection.create(tr.doc, transaction.mapping.maps[0].ranges[0] + 1)
```

Comment:
Thanks for writing this PR @doodlewind.

Apologies if I'm missing something obvious, but is `transaction.mapping...` a typo? Where is `transaction` defined? Should that be `tr.mapping.maps[0]` instead?

Furthermore, for many instances it seems that `tr.mapping.maps` is an empty array - causing the function call to fail.